### PR TITLE
cn: make functions to simplify cn.New()

### PR DESCRIPTION
**Pull request by ehnuje**
## Proposed changes

- To simplify `cn.New()` to test it, some of the code lines are extracted to functions
- Also, as `chainConfig` and `cn.chainConfig` are identical, only use `cn.chainConfig` to remove ambiguity

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: node/cn/backend.go [hunk 1]
```diff
@@ -259,28 +259,14 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
...
-		istanbulExtra, err := types.ExtractIstanbulExtra(cn.blockchain.Genesis().Header())
-		if err != nil {
-			logger.Error("Failed to decode IstanbulExtra", "err", err)
-		} else {
-			if len(istanbulExtra.Validators) == 1 {
-				atomic.StoreUint32(&cn.protocolManager.acceptTxs, 1)
-			}
-		}
+	if err := cn.setAcceptTxs(); err != nil {
+		logger.Error("Failed to decode IstanbulExtra", "err", err)
```
Comment (nogang) [inline node/cn/backend.go]: Wouldn't it be better to do this in a function that prints an error? Do it need to get an error and print it back out?

File: node/cn/backend.go [hunk 2]
```diff
@@ -259,28 +259,14 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
...
-	if cn.protocolManager.nodetype == node.CONSENSUSNODE {
-		wallet, err := cn.RewardbaseWallet()
-		if err != nil {
-			logger.Error("find err", "err", err)
-		} else {
-			cn.protocolManager.SetRewardbaseWallet(wallet)
-		}
-		cn.protocolManager.SetRewardbase(cn.rewardbase)
+	if err := cn.setRewardWallet(); err != nil {
+		logger.Error("find err", "err", err)
```
Comment (nogang) [inline node/cn/backend.go]: It's too

File: node/cn/backend.go (hunk 1 above)
Comment (ehnuje) [reply node/cn/backend.go]: You're right. However, if we want to test a function, we want to know whether it returns an error when expected.

File: node/cn/backend.go (hunk 2 above)
Comment (ehnuje) [reply node/cn/backend.go]: Same comment.

File: node/cn/backend.go [hunk 3]
```diff
@@ -312,6 +298,35 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 	return cn, nil
 }
 
+// setAcceptTxs sets AcceptTxs flag in 1CN case to receive tx propagation.
+func (s *CN) setAcceptTxs() error {
```
Comment (KimKyungup) [inline node/cn/backend.go]: How about this?
```suggestion
func (s *CN) setAcceptTxsFor1CN() error {
```

Comment (ehnuje) [reply node/cn/backend.go]: `For1CN` sounds a little bit ambiguous. Let me merge this for now and let's change the name after if it still does not look proper.

**Pull request merged**